.gitignore

```diff
@@ -1,5 +1,5 @@
 /target
-
+.idea # IntelliJ IDEA
```

Comment:
your whole .idea is still committed, please run this: https://stackoverflow.com/questions/7927230/remove-directory-from-remote-repository-after-adding-them-to-gitignore

also, in the future, make a Draft PR, then go to "files changed" and check that everything's ready to go, and then tag a reviewer please! (helps with notification spam)

Reply:
removed. sorry about that.